doc/hopauth_guide.md

```diff
@@ -74,112 +77,130 @@ Logging in goes through the CILogon service and your institutional single sign o
+
+## Manage Groups and Topics
+
+If you are an owner of a group, you have the authority to manage the membership of that group. All groups to which you belong are listed on the main page, and for each group in which you are an owner, there will be a button to 'Manage Users' and another to 'Manange Topics'. After clicking, each button will take you to the proper management page.
+
+[![Main page group's section](hopauth_screenshots/group_management_main.png)](hopauth_screenshots/group_management_main.png)
 
 ### Managing Group Members
 
-If you are an owner of a group, you have the authority to manage the membership of that group. All groups to which you belong are listed on the main page, and for each group in which you are an owner, there will be an 'Edit' button which will take you to the management page for that group. 
+The management page for a group shows all current members and their level of membership, gives the option to add new group members or remove the existing ones.
 
 [![The management page for a group](hopauth_screenshots/group_management.png)](hopauth_screenshots/group_management.png)
 
-The management page for a group shows all current members and their level of membership, gives the option to add new group members, lists topics owned by the group, and gives the option to create new topics. 
+To add a member, simply click the 'Add' button for any user in the list at the 'Available Users' section. After adding it to your group, you can select whether that person should be a regular member or also be and owner from the drop-down menu under 'Permission'. Users are listed with both their internal SCiMMA identifier and their email address, which is easier to recognize.
 
-To add a member, simply select the user from the drop-down list in the 'Add Member' section, select whether that person should be a regular member or also be and owner, and then click the 'Add user' button. Users are listed with both their internal SCiMMA identifier and their email address, which is easier to recognize. 
+In the 'Current Members' section of the page any member of the group can be removed or have their membership status switched between regular membership and ownership of the group.
 
-In the 'Current Members' section of the page any member of the group can be removed or have their membership status switched between regular membership and ownership of the group. 
+### Managing Topics
 
-### Creating Topics
+If you are the owner of a group you can create topics owned by your group where you and other members of your group can write (and read) data in Hopskotch. To do so, enter the desired topic name into the field in the 'Create Topic' section of the group topic's management page, and click the 'Create' button. Topic names must be unique across the system, may be 1-249 characters long, and may only contain the characters `a-zA-Z0-9._-`.
```

Comment:
I think we can replace 'across the system' with 'within the group', as we now enforce all topics being prefixed with the group name. (So, they are technically still globally unique, but users don't have to directly worry about that.)